app/src/js/server/util.ts

```diff
@@ -81,7 +81,15 @@ function validateCognitoConfig (cognito: CognitoConfig | undefined) {
  *
  * @param {ServerConfig} config
  */
-function validateConfig (config: ServerConfig) {
+async function validateConfig (config: ServerConfig) {
+  if (!(await fs.pathExists(config.data))) {
+    throw new Error(`Cannot find ${config.data}`)
+  }
+
+  if (!(await fs.pathExists(config.itemDir))) {
```

Comment:
`itemDir` is optional

Reply:
This check happens after the config is supplemented by the default config, so `itemDir` will always have a value here. Don't we need to verify that the dir exists? Otherwise there may be an error when saving later.

Reply:
It may not exist if the user doesn't specify it. It is not necessary unless the the user wants to server local data.

Reply:
So we should only check the itemDir if the database is local? In the case where database is local but itemDir is not specified, should we throw an error that the default itemDir is missing?

Reply:
We can set the default itemDir to empty string. We only check whether the itemDir exists when local FSU’s used and itemDir is not empty.

Reply:
Fixed